### PR TITLE
tests: Add package metadata

### PR DESCRIPTION
**Pull request by jcs090218**
Few improvements for CI.

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -16,19 +16,15 @@ jobs:
...
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - uses: jcs090218/setup-emacs@master
         with:
           version: ${{ matrix.emacs-version }}
 
-      - uses: actions/setup-node@v2
-        with:
-          node-version: '14'
```
Comment (jcs090218) [inline .github/workflows/build.yaml:30]: `setup-node` is no longer required.

**Comment by jojojames** [comment]
All the checks are failing.

**Comment by jcs090218** [comment]
The CI fails on the master branch due to the latest commit 0fbfcbace0729dba685da9b9a74566639b840ec3. Not sure why silencing the byte compiler will cause the to test fail though. 😕

**Comment by jojojames** [comment]
What's the right way to handle this?

**Comment by jcs090218** [comment]
It seems like one test fail,

```
Test fussy-pattern-flex-2-test backtrace:
  signal(ert-test-failed (((should (string= (fussy-pattern-str (fussy-
  ert-fail(((should (string= (fussy-pattern-str (fussy-pattern-flex-2 
  (if (unwind-protect (setq value-322 (apply fn-320 args-321)) (setq f
  (let (form-description-324) (if (unwind-protect (setq value-322 (app
  (let ((value-322 'ert-form-evaluation-aborted-323)) (let (form-descr
  (let* ((fn-320 #'string=) (args-321 (condition-case err (let ((signa
  (let ((orderless-matching-styles '(orderless-flex))) (let* ((fn-315 
  (closure (t) nil (let ((orderless-matching-styles '(orderless-flex))
  ert--run-test-internal(#s(ert--test-execution-info :test #s(ert-test
  ert-run-test(#s(ert-test :name fussy-pattern-flex-2-test :documentat
  ert-run-or-rerun-test(#s(ert--stats :selector t :tests ... :test-map
  ert-run-tests(t #f(compiled-function (event-type &rest event-args) #
  ert-run-tests-batch(nil)
  ert-run-tests-batch-and-exit()
  (let ((eask--ignore-error-p t)) (mapc #'load-file files) (ert-run-te
  (progn (eask-pkg-init) (let ((eask--ignore-error-p t)) (mapc #'load-
  (if files (progn (eask-pkg-init) (let ((eask--ignore-error-p t)) (ma
  (let* ((files (and t (eask-expand-file-specs (eask-args))))) (if fil
  (progn (run-hooks 'eask-before-command-hook) (run-hooks (intern (con
  (progn (if eask-file (eask-msg "✓ Loading Eask file in %s... done!" 
  (if (or (eask-file-try-load "../../") special) (progn (if eask-file 
  (let* ((user-emacs-directory (expand-file-name (concat ".eask/" emac
  (cond ((eask-global-p) (if (eask-file-try-load "./") (eask-msg "✓ Lo
  (let ((default-directory (if (eask-global-p) user-emacs-directory de
  (let ((argv (eask-args)) load-file-name buffer-file-name) (let ((def
  (if eask--initialized-p (progn (let* ((files (and t (eask-expand-fil
  (if eask-loading-file-p nil (if eask--initialized-p (progn (let* ((f
  eval-buffer(#<buffer  *load*> nil "/home/runner/eask-0.7.4/lisp/test
  load-with-code-conversion("/home/runner/eask-0.7.4/lisp/test/ert.el"
  load("/home/runner/eask-0.7.4/lisp/test/ert.el" nil t t)
  command-line-1(("-scriptload" "/home/runner/eask-0.7.4/lisp/test/ert
  command-line()
  normal-top-level()
Test fussy-pattern-flex-2-test condition:
    (ert-test-failed
     ((should
       (string=
	(fussy-pattern-str ...)
	(fussy-pattern-str ...)))
      :form
      (string= "(\"\\\\(?:\\\\(?:\\\\(a\\\\).*\\\\(b\\\\).*\\\\(c\\\\)\\\\)\\\\)\")" "(\"\\\\(?:\\\\(?:\\\\(a\\\\)[^b]*\\\\(b\\\\)[^c]*\\\\(c\\\\)\\\\)\\\\)\")")
      :value nil))
   FAILED  13/23  fussy-pattern-flex-2-test (0.0004[64](https://github.com/jojojames/fussy/actions/runs/3665308253/jobs/6196292291#step:10:65) sec)
```

I have cloned and tested, and whether the commit [0fbfcba](https://github.com/jojojames/fussy/commit/0fbfcbace0729dba685da9b9a74566639b840ec3) is made or not, the result does not change (error appears). I suspect there are breaking changes from the upstream package orderless.

**Comment by jojojames** [comment]
Is this still the right patch @jcs090218 ? Happy to merge if so.

**Comment by jcs090218** [comment]
Yeah, the tests are irrelevant to the changes. I'll merge this now! :)

**Pull request merged**